### PR TITLE
Aligned the reset button to the reset text

### PR DESCRIPTION
**Pull request by mariacloehb**

### CODE REVIEW

File: frontend/scss/vendor/mathigon.scss [hunk 1]
```diff
@@ -240,11 +240,14 @@ x-progress {
...
   }
 
+
+
   path.pie {
     color: $background-color-tertiary;
   }
 }
 
+
```
Comment (eddybrando) [inline frontend/scss/vendor/mathigon.scss:250]: I believe this wasn't meant to be included:

```suggestion
```

File: frontend/scss/vendor/mathigon.scss [hunk 2]
```diff
@@ -240,11 +240,14 @@ x-progress {
     position: relative;
   }
 
+
+
```
Comment (eddybrando) [inline frontend/scss/vendor/mathigon.scss:244]: I believe this wasn't meant to be included:

```suggestion
```

File: frontend/vue/components/MiniComposer/CircuitSandboxWidget.vue [hunk 3]
```diff
@@ -51,7 +52,7 @@
       <section>
         <h1 class="mini-composer__state-views__title">
           State Vector
-        </h1>
+        </h1>rm
```
Comment (eddybrando) [inline frontend/vue/components/MiniComposer/CircuitSandboxWidget.vue:55]: I believe this wasn't meant to be included:

```suggestion
        </h1>
```

File: frontend/vue/components/MiniComposer/CircuitSandboxWidget.vue [hunk 4]
```diff
@@ -24,7 +24,8 @@
         class="mini-composer__circuit-section__reset"
         @click="onResetButton"
       >
-        Reset <PlayIcon />
+
+      Reset <ResetIcon class="mini-composer__circuit-section__reset__icon" />
```
Comment (eddybrando) [inline frontend/vue/components/MiniComposer/CircuitSandboxWidget.vue:28]: Here we need to remove the empty line and add some indentation:

```suggestion
        Reset <ResetIcon class="mini-composer__circuit-section__reset__icon" />
```

File: frontend/vue/components/MiniComposer/CircuitSandboxWidget.vue [hunk 5]
```diff
@@ -65,7 +66,7 @@
 import { ref } from '@vue/reactivity'
 import { Options, Vue, prop } from 'vue-class-component'
 import draggable from 'vuedraggable'
-import PlayIcon from '@carbon/icons-vue/lib/reset/16'
+import ResetIcon from '@carbon/icons-vue/lib/reset/16'
```
Comment (eddybrando) [inline frontend/vue/components/MiniComposer/CircuitSandboxWidget.vue:69]: Nice catch 👍

**Review by eddybrando — CHANGES_REQUESTED** [review]
Thanks a lot for your first PR @mariacloehb! 🙂

I added a couple of comments.

We need to get rid of changes that are not relevant to the PR. I added comments on those.

I liked the renaming of the icon component. It's a good practice to fix this kind of stuff as we see it.

To solve the alignment issue, there is a simpler approach: We can just add `align-items: center` to the `mini-composer__circuit-section__reset` CSS class. That way, we. don't need to make the height and width adjustments.

I strongly recommend this resource to leverage the power of flexbox: https://css-tricks.com/snippets/css/a-guide-to-flexbox/

File: frontend/scss/vendor/mathigon.scss [hunk 6]
```diff
@@ -245,6 +245,7 @@ x-progress {
   }
 }
 
+
```
Comment (eddybrando) [inline frontend/scss/vendor/mathigon.scss:248]:
```suggestion
```

File: frontend/vue/components/MiniComposer/CircuitSandboxWidget.vue [hunk 7]
```diff
@@ -273,8 +276,8 @@ export default class CircuitSandboxWidget extends Vue.with(Props) {
       align-items: center;
       justify-content: center;
 
-      height: 9rem;
-      width: 100%;
+      height: 16px;
```
Comment (techtolentino) [inline frontend/vue/components/MiniComposer/CircuitSandboxWidget.vue:279]: One question I had here — I'm wondering if these values needed to change?

Updating the `height` and `width` caused the formula area to be too small. However, if we leave them at `height: 9rem` and `width: 100%`, the formula is readable.

Please see this screen capture in Firefox 👇🏼 



https://user-images.githubusercontent.com/6276074/191624451-5f3a5e2e-609b-4185-a04e-fdd8ee2c1750.mov

**Review by techtolentino — CHANGES_REQUESTED** [review]
The reset alignment looks great, nicely done 👏🏼 

Just leaving a comment about whether or not the `mini-composer__state-views__area` height and width needed updating.

Thank you!

File: frontend/vue/components/MiniComposer/CircuitSandboxWidget.vue [hunk 8]
```diff
@@ -276,8 +276,8 @@ export default class CircuitSandboxWidget extends Vue.with(Props) {
       align-items: center;
       justify-content: center;
 
-      height: 16px;
-      width: 16px;
+      height: 9rem;
```
Comment (techtolentino) [inline frontend/vue/components/MiniComposer/CircuitSandboxWidget.vue:279]: @mariacloehb - this should just be 1rem... 

16px = 1rem

Comment (techtolentino) [reply frontend/vue/components/MiniComposer/CircuitSandboxWidget.vue:279]: wait, disregard...

**Review by techtolentino — APPROVED** [review]
Nicely done 👏🏼 

LGTM

**Review by vabarbosa — APPROVED** [review]
thank you!!

**Pull request merged**